src/Supportive/DependencyInjection/Configuration.php

```diff
@@ -207,8 +207,10 @@ private function appendEmitterTypes(ArrayNodeDefinition $node): void
                 ->arrayNode('analyser')
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->scalarNode('internal_tag')
+                            ->defaultNull()
```

Comment:
isn't the default `@internal`?

Reply:
We could do it either way... I thought it would be more intuitive if the default was to use just @deptrac-internal, and allow an additional tag to be specified in the config. But that would be a change in behavior. 

Let me know which you prefer.

Reply:
`@deptrac-internal` always, plus optionally another one, letting the `internal_tag` option default to `@internal` (This is not BC change and can be released in minor release)

Reply:
As mentioned earlier, I went for the breaking change, since @gennadigennadigennadi mentioned a version bump is planned anyway.

By the way... "you use BC" to mean "breaking change", I am used to "B/C" meaning "backwards compatible"... Let's be careful not to get confused :)

Reply:
Right. OK, let's call it a change that requires a major just as per SemVer. :)